styles/themes/_dark.scss

```diff
@@ -89,6 +96,15 @@ html.dark-mode {
       }
     }
 
+    .navbar__sign-out-button {
+      border-color: $white !important;
+      box-shadow: 4px 4px 0px $white !important;
+      background: $dark-accent !important;
+      color: $white !important;
+      &:hover {
+        background-color: $primary !important;
+      }
+    }
```

Comment:
There are already two different theme styles for `sign-out-button`. So pls remove these lines to `mian-navigation.scss` file and take care not to break the style on light mode.
<img width="348" alt="image" src="https://user-images.githubusercontent.com/31397967/211180775-8877044f-ca85-4ea8-9348-582f3656f9df.png">

Reply:
where is this used? I cant find any class called sign-out-button, I think we can remove this css altogether

Reply:
Yup, then let's move it.